### ISSUE TITLE
Deprecate 'Choices'

### ISSUE THREAD

**Issue by adamchainz** [jazzband/django-model-utils#422]
## Problem

Django 3.0 adds enumeration types, which perform the same function as the `Choices` class. It's best we recommend users use the core django feature rather than duplicating it

PR #421 added a note about this.

At least once the minimum Django version is 3.0, we should deprecate `Choices` and remove it some time after.

See also [my blog post](https://adamj.eu/tech/2020/01/27/moving-to-django-3-field-choices-enumeration-types/).

**Comment by lifelonglearner127** [comment]
Yes. But Django 3.0 does not support named group, whereas `django-model-utils` does.

**Comment by adamchainz** [comment]
Yes that's fair. There was a discussion on django-developers to add support, it should come in a future Django version I expect.
